libs/template/config.go

```diff
@@ -273,3 +274,23 @@ func (c *config) validate() error {
 	}
 	return nil
 }
+
+// Return enum values selected by the user during template initialization. These
+// values are safe to send over in telemetry events due to their limited cardinality.
+func (c *config) enumValues() map[string]string {
+	res := map[string]string{}
+	for k, p := range c.schema.Properties {
+		if p.Type != jsonschema.StringType {
+			continue
+		}
+		if p.Enum == nil {
+			continue
+		}
+		v := c.values[k]
+
+		if slices.Contains(p.Enum, v) {
```

Comment:
This is not strictly necessary since we'd error earlier during schema validation, but this added to have an additional guardrail against logging PII.